Core/Rewriters/Formatter.cs

```diff
@@ -136,32 +136,23 @@ private static ExpressionStatementSyntax RemoveRedundantWhitespaces (ExpressionS
 
     private static InvocationExpressionSyntax FormatInvocationExpression (InvocationExpressionSyntax node)
     {
-      var indentation = node.Expression.GetIndentation();
       return node.Expression is not MemberAccessExpressionSyntax memberAccessExpression
           ? node
-          : node.WithExpression (FormatMemberAccessExpression (memberAccessExpression, indentation)).WithoutTrailingTrivia();
+          : node.WithExpression (FormatMemberAccessExpression (memberAccessExpression)).WithoutTrailingTrivia();
     }
 
-    private static MemberAccessExpressionSyntax FormatMemberAccessExpression (MemberAccessExpressionSyntax node, string indentation)
+    private static MemberAccessExpressionSyntax FormatMemberAccessExpression (MemberAccessExpressionSyntax node)
     {
+      var indentation = ((MemberAccessExpressionSyntax) node.DescendantNodes().First (s => s.IsKind (SyntaxKind.SimpleMemberAccessExpression))).OperatorToken.LeadingTrivia.ToString();
       return MoqSyntaxFactory.MemberAccessExpression (
           node.Expression.WithTrailingTrivia (SyntaxFactory.Whitespace (Environment.NewLine + indentation)),
           node.Name);
     }
 
     private static bool IsMultiLineStatement (SyntaxNode node)
     {
-      var nodes = node.DescendantNodes().Where (
-          s => s.IsKind (SyntaxKind.SimpleMemberAccessExpression)
-               && !s.Ancestors().Any (s => s.IsKind (SyntaxKind.Argument))).Select (s => (MemberAccessExpressionSyntax) s);
-
-      return nodes.Any (
-          s => s.Expression.GetTrailingTrivia().ToFullString().Contains (Environment.NewLine)
-               || s.Expression.GetLeadingTrivia().ToFullString().Contains (Environment.NewLine)
-               || s.Name.GetTrailingTrivia().ToFullString().Contains (Environment.NewLine)
-               || s.Name.GetLeadingTrivia().ToFullString().Contains (Environment.NewLine)
-               || s.OperatorToken.TrailingTrivia.ToFullString().Contains (Environment.NewLine)
-               || s.OperatorToken.LeadingTrivia.ToFullString().Contains (Environment.NewLine));
+      return node.GetFirstIdentifierName().GetTrailingTrivia().ToFullString().Contains (Environment.NewLine) ||
+             (node.GetFirstIdentifierName().Parent!.Parent! as InvocationExpressionSyntax)?.ArgumentList.GetTrailingTrivia().ToFullString().Contains (Environment.NewLine) == true;
```

Comment:
what about the following:
```suggestion
      var nodeLineSpan = node.GetLocation().GetLineSpan();
      return nodeLineSpan.StartLinePosition.Line != nodeLineSpan.EndLinePosition.Line;
```

Reply:
no this would not result in the same beahaviour e.g. even tough this statement stretches over multiple lines it should not be formatted
```
mock.Setup (_ => _.DoSomething (
      1,
      2)).Returns (3).Verifiable();
``` 
      
however this statement should be formatted, because it is a multiline statement      

```
mock.Setup (_ => _.DoSomething (
      1,
      2))
      .Returns (3).Verifiable();
```

Reply:
@MichaelKetting how should this be done?